documents/validator.go

```diff
@@ -400,6 +400,33 @@ func anchorRepoAddressValidator(anchoredRepoAddr common.Address) Validator {
 	})
 }
 
+// attributeValidator validates the signed attributes.
+// TODO(ved): tests
+func attributeValidator(idSrv identity.Service) Validator {
```

Comment:
I assume you will still add this to the validator groups?

Reply:
not complete yet